### ISSUE TITLE
Codex Inquisition / Adepta Sororitas / Imperial Agents

### ISSUE THREAD

**Issue by Viper666-Qc** [BSData/wh40k-7th-edition#3296]
I think you'll have to add an option in the Adepta Sororitas and Inquisition rosters to make the units appear or disappear depending you use the army list from their respective codex or the one from the Codex Imperial Agents. 

Right now, you can mix and match those units without restriction and I think it is not intented to be like that. GW stated that both publication (AS or Inq Codex and Imperial Agents) are still legal but they still are distinctive army lists on their own.

For example, right now I can have a Inquisition detachment with a Ordo Xenos inquisitor (Inq) AND a Ordo Xenos Inquisitor (C:IA) at the same time with different rules and options... (by the way, ther CAD and Allied Detachment option for the Inquisition roster shouldn't appear since they don't have Troop options).

Same for a Adepta Sororitas detachment where you can add an Ecclesiarchy Battle Conclave AND Assassins/Crusaders/Arco-Flagellants at the same time.....

Finally, if using the Codex: Adepta Sororitas, Ministorum Priests should still be available as a No Force Org slot (0-5) rather than HQ slot (as in the C:IA army list entry).

Bottom Line: GW are totally stupid to make 2 valid sources of datasheets for the same faction....

**Comment by Viper666-Qc** [comment]
And this will become even more ugly with the Castellans of the Imperium detachment.....

**Comment by alphalas** [comment]
They have the same faction - so yes it's dumb, but I do believe that is exactly what GW intended. Only change I think I need to make is link a no FOC priest as well as the HQ priest

**Comment by alphalas** [comment]
If GW intended to be an Either/Or, they wouldn't have the same faction

**Comment by Viper666-Qc** [comment]
then it would also be possible to include a Inq. Coteaz and a C:IA Coteaz at the same time since they get the same faction but different valid datasheets (and different rules)....

**Comment by alphalas** [comment]
I would say no - Cassius has 3 profiles, and you can only use one at one time.

**Comment by khelemvor** [comment]
The recent [FAQ for Imperial Agents](https://www.warhammer-community.com/wp-content/uploads/2017/01/Imperial_Agents_v1.1.pdf) that the old digital codexes along with the profiles in Imperial Agents are both valid options. 

> Q: Imperial Agents seems to replace existing material but
> Games Workshop has indicated that players can use either
> the new or older material. As Imperial Agents lacks certain
> things present in older material, such as Servo Skulls, which do
> we use?
> A: If you have bought and own the existing and still
> currently available digital Codex: Adepta Sororitas and
> Codex: Inquisition, you can use those over the rules
> presented in Imperial Agents if you wish.

**Comment by alphalas** [comment]
Yes, that is an established fact, thank you.

**Comment by alphalas** [comment]
If anyone would like to further discuss this, please take it to Gitter.
